java/README.md

```diff
@@ -3,6 +3,7 @@ Java Rendler Framework:
 <br> 1) cd hostfiles/java </br> 
 <br> 2) mvn clean compile assembly:single </br>
 <br> 3) export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib </br>
-<br> 4) java -cp target/rendler-1.0-SNAPSHOT-jar-with-dependencies.jar RendlerMain 127.0.1.1:5050 250 </br>
+ <br> # Run RendlerMain with parameter 'master' or parameters 'master' 'tasks' 'url'. In former, default values for tasks and url are 5 and "http://mesosphere.com", respectively.</br>
+<br> 4) java -cp target/rendler-1.0-SNAPSHOT-jar-with-dependencies.jar RendlerMain 127.0.1.1:5050</br>
```

Comment:
This doesn't reference any of the parameters you mention in the line above. I would remove `In former, default values for tasks and url are 5 and "http://mesosphere.com", respectively.` and just put them in this example command itself.